syntax = "proto2";

package opentxs.proto;
option optimize_for = LITE_RUNTIME;

import public "StorageItemHash.proto";

message StorageNym {
    optional uint32 version = 1;
    optional string nymid = 2;
    optional StorageItemHash credList = 3;
}
